packages/twenty-front/src/modules/activities/files/components/DropZone.tsx

```diff
@@ -0,0 +1,121 @@
+import { useDropzone } from 'react-dropzone';
+import styled from '@emotion/styled';
+
+import { useSpreadsheetImportInternal } from '@/spreadsheet-import/hooks/useSpreadsheetImportInternal';
+import { IconUpload } from '@/ui/display/icon';
+
+const StyledContainer = styled.div`
+  align-items: center;
+  background-position:
+    0 0,
+    0 0,
+    100% 0,
+    0 100%;
+  background-repeat: no-repeat;
+  background-size:
+    2px 100%,
+    100% 2px,
+    2px 100%,
+    100% 2px;
+  border-radius: ${({ theme }) => theme.border.radius.md};
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  height: 100%;
+  border: ${({ theme }) => `2px dashed ${theme.border.color.strong}`};
+`;
+
+const StyledOverlay = styled.div`
+  background: ${({ theme }) => theme.background.secondary};
+  border-radius: ${({ theme }) => theme.border.radius.sm};
+  bottom: 0px;
+  left: 0px;
+  position: absolute;
+  right: 0px;
+  top: 0px;
+`;
```

Comment:
Please correct me if I'm wrong, but I think `StyledOverlay` is not necessary and can be merged with `StyledContainer`? Here's what I thought we could do:

```suggestion
const StyledContainer = styled.div`
  align-items: center;
  background: ${({ theme }) => theme.background.secondary};
  border: ${({ theme }) => `2px dashed ${theme.border.color.strong}`};
  border-radius: ${({ theme }) => theme.border.radius.md};
  display: flex;
  flex-direction: column;
  height: 100%;
  justify-content: center;
  border-radius: ${({ theme }) => theme.border.radius.sm};
  text-align: center;
`;
```
Then we can remove all the `z-index: 1`!